target/linux/generic/hack-5.4/645-netfilter-connmark-introduce-set-dscpmark.patch

```diff
@@ -87,8 +87,8 @@ Signed-off-by: Kevin Darbyshire-Bryant <ldir@darbyshire-bryant.me.uk>
  };
  
  enum {
-+	XT_CONNMARK_VALUE = BIT(0),
-+	XT_CONNMARK_DSCP = BIT(1)
++	XT_CONNMARK_VALUE = (1 << 0),
++	XT_CONNMARK_DSCP = (1 << 1)
```

Comment:
FYI that original BIT() macro translates into `#define BIT(nr) (UL(1) << (nr))`.